distribution/lib/Standard/Google_Api/0.0.0-dev/src/Sheets/Google_Sheets.enso

```diff
@@ -0,0 +1,64 @@
+from Standard.Base import all
+import Standard.Base.Data.Array_Proxy.Array_Proxy
+import Standard.Base.Metadata.Display
+from Standard.Base.Metadata import make_single_choice, Widget
+from Standard.Base.Metadata.Widget import Single_Choice, Vector_Editor
+from Standard.Table import Table
+
+polyglot java import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+polyglot java import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+polyglot java import com.google.api.client.json.gson.GsonFactory
+polyglot java import com.google.api.services.sheets.v4.Sheets
+polyglot java import com.google.api.services.sheets.v4.SheetsScopes
+polyglot java import java.util.Collections
+
+
+## PRIVATE
+type Google_Sheets
+    ## PRIVATE
+    Value credential json_factory http_transport
+
+    ## Accesses a service responsible for working with Google Spreadsheets.
+
+       Arguments:
+       - app_name: the application name to use for making the API calls. This
+         will show up in access logs etc.
+    spreadsheets : Text -> Spreadsheets
+    spreadsheets self app_name='Enso' =
+        service = Sheets.Builder.new self.http_transport self.json_factory self.credential . setApplicationName app_name . build
+        Spreadsheets.Service service
```

Comment:
It is indeed a bit confusing to me that I call `Google_Sheets.spreadsheets` to get an instance of `Spreadsheets` to work with.

What is the difference between `Google_Sheets` and `Spreadsheets`? They both seem to mean _exactly the same thing_.

Why can't we merge the two types into one and initialize them in one go within `initialize`?